src/db/sequelize.js

```diff
@@ -211,6 +211,26 @@ function initManually(database, username, password, options) {
   Invite.belongsTo(User, { as: 'userInvited' });
   Invite.belongsTo(Project, { as: 'projectInvitedTo' });
 
+  const GithubToken = model.define('githubToken', {
+    id: {
+      primaryKey: true,
+      type: UUID,
+      defaultValue: UUIDV4,
+    },
+    token: STRING(2000)
+  });
+
+  const GithubTokenOwner = model.define('githubTokenOwner', {
+    // ASSOCIATIONS DEFINED BELOW
+    id: {
+      primaryKey: true,
+      type: UUID,
+      defaultValue: UUIDV4,
+    }
+  });
+  GithubToken.belongsToMany(User, { as: 'owner', through: 'githubTokenOwner' });
+  User.belongsToMany(GithubToken, { as: 'githubToken', through: 'githubTokenOwner' });
```

Comment:
Shouldn't this be a one-one relationship rather than a many-many??

Reply:
Yes it should. But I couldn’t get it to work as sequelize would always give me a foreign key constraint error

Reply:
Did you try adding the foreign keys in the models?

Reply:
Yup 🙁